src/Styles/base.scss

```diff
@@ -12,10 +12,68 @@
 
 body, html {
   height: 100%;
-
+  font-family: 'Poppins', sans-serif;
 }
 
 h1,
 h2 {
   font-family: 'Fredoka One', cursive;
 }
+
+h3,
+p,
+label,
+button,
+textarea,
+input,
+::placeholder {
+  font-family: inherit;
+}
+
+textarea {
+  resize: none;
+  width: 99%;
+}
+
+.modal {
```

Comment:
This was a great learning on how to control the styling for the application.  Feels much more organized.